lametro/models.py

```diff
@@ -141,21 +142,9 @@ def inferred_status(self):
         return self._status(description)
 
     def _status(self, description):
-        if description:
-            if description.upper() in ['APPROVED', 'APPROVED AS AMENDED', 'APPROVED THE CONSENT CALENDAR']:
-                return 'Approved'
-            elif description.upper() in ['ADOPTED', 'ADOPTED AS AMENDED']:
-                return 'Adopted'
-            elif description.upper() in ['RECOMMENDED FOR APPROVAL', 'RECOMMENDED FOR APPROVAL AS AMENDED', 'REFERRED', 'FORWARDED DUE TO ABSENCES AND CONFLICTS', 'FORWARDED WITHOUT RECOMMENDATION', 'NO ACTION TAKEN', 'NOT DISCUSSED']:
-                return 'Active'
-            elif description.upper() in ['RECEIVED', 'RECEIVED AND FILED']:
-                return 'Received'
-            elif description.upper() == 'FAILED':
-                return 'Failed'
-            elif description.upper() == 'WITHDRAWN':
-              return 'Withdrawn'
-        else:
-            return None
+        if description and BILL_STATUS_DESCRIPTIONS[description.upper()]:
```

Comment:
What happens if the specified description is not in the descriptions dictionary?

Reply:
my understanding, based on the logic of the previous _status method, was that the status should be None (line 158 in the previous version, line 147 in the updated code). did I miss something?

Reply:
More precisely, what happens if you try to access a key that does not exist in a dictionary in this way?

Reply:
oh, gotcha, it'll throw an error rather than considering the condition false. I'll revise the logic. thanks!